recipe-client-addon/lib/StudyStorage.jsm

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @typedef {Object} Study
+ * @property {string} name
+ *   Unique name of the study
+ * @property {number} version
```

Comment:
@Osmose Can you confirm what the `version` type will be? I imagined using ints since we don't support major/minor revisions to recipes, just full revisions.